app/src/main/kotlin/co/netguru/baby/monitor/client/common/NotificationHandler.kt

```diff
@@ -1,53 +1,55 @@
 package co.netguru.baby.monitor.client.common
 
-import android.app.*
+import android.app.Notification
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/feb0c51d1649c186c776faa50e562320ab9dc395/app/src/main/kotlin/co/netguru/baby/monitor/client/common/NotificationHandler.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>